drivers/entropy/entropy_renesas_ra.c

```diff
@@ -35,15 +33,27 @@ static int entropy_ra_rsip_trng_get_entropy(const struct device *dev, uint8_t *b
 	return 0;
 }
 
-static const struct entropy_driver_api entropy_ra_rsip_trng_api = {
-	.get_entropy = entropy_ra_rsip_trng_get_entropy,
+static const struct entropy_driver_api entropy_renesas_ra_api = {
```

Comment:
```suggestion
static DEVICE_API(entropy, entropy_renesas_ra_api) = {
```
https://github.com/zephyrproject-rtos/zephyr/pull/82174/files

Reply:
Done